AR1/reduction/interferometric_pointing/analyse_interferometric_pointing.py

```diff
@@ -118,8 +119,8 @@ def _eq_TGT_(tgt): # tgt==TGT, "tags" don't matter
         gains_p[pol] = []
         pos = []
         stdv[pol] = []
-        h5.select(pol=pol,corrprods='cross',ants=active_ants)
-        h5.bls_lookup = calprocs.get_bls_lookup(active_ants,h5.corr_products)
+        h5.select(pol=pol,corrprods='cross',ants=antlist,targets=[h5.catalogue.targets[TI] for TI in target_indices],compscans=compscan_index)
+        bls_lookup = calprocs.get_bls_lookup(antlist,h5.corr_products)
```

Comment:
This line is the only line that really matters... (the change from `active_ants` to `antlist` is the crux).

Reply:
The adding of targets & comps cans to the selection is a bit redundant .

Reply:
I agree, let's remove the redundancy then...